x-pack/plugins/secops/public/components/timeline/body/renderers/primary_secondary_user_info.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { DraggableBadge } from '../../../draggables';
+
+const TokensFlexItem = styled(EuiFlexItem)`
+  margin-left: 3px;
+`;
+
+export const nilOrUnSet = (value?: string | null) =>
+  value == null || value.toLowerCase() === 'unset';
+
+export const PrimarySecondary = pure<{
+  id: string;
+  primary?: string | null;
```

Comment:
should `primary` and `secondary` be required props (applies to multiple files) in this PR?

Reply:
I will make them required, yea.